deepchem/molnet/load_function/tests/tests_bbbc.py

```diff
@@ -0,0 +1,92 @@
+"""
+Tests for BBBC Loaders.
+"""
+
+import unittest
+import deepchem as dc
+
+
+class TestBBBCLoader(unittest.TestCase):
```

Comment:
Let's move this from the current PR to a separate PR where we can put into a custom CI workflow for moleculenet that runs weekly